#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_remove_vlc_cable.dpatch by  <scott@kitterman.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix FTBFS with newer libs - Cherry pick from upstream svn
## DP: http://svn.tilp.info/cgi-bin/viewvc.cgi/tiemu/trunk/src/gui/device.c?root=tiemu&view=log#rev2820

@DPATCH@
diff -urNad tiemu-3.02~/src/gui/device.c tiemu-3.02/src/gui/device.c
--- tiemu-3.02~/src/gui/device.c	2007-12-30 18:14:15.000000000 -0500
+++ tiemu-3.02/src/gui/device.c	2010-04-26 01:34:36.657234841 -0400
@@ -102,10 +102,6 @@
 		gtk_option_menu_set_history(GTK_OPTION_MENU(data), 7);
 	break;
 	
-	case CABLE_VTL:
-		gtk_option_menu_set_history(GTK_OPTION_MENU(data), 8);
-	break;   
-
 	case CABLE_ILP:
 		gtk_option_menu_set_history(GTK_OPTION_MENU(data), 9);
 	break;
@@ -194,8 +190,7 @@
 	case 5: tmp.cable_model = CABLE_USB; break;	
 	case 6: tmp.cable_model = CABLE_VTI; break;
 	case 7: tmp.cable_model = CABLE_TIE; break;
-	case 8: tmp.cable_model = CABLE_VTL; break;
-	case 9: tmp.cable_model = CABLE_ILP; break;
+	case 8: tmp.cable_model = CABLE_ILP; break;
 	}
 }
 
